assets/js/blocks/customer-account/sidebar-settings.tsx

```diff
@@ -0,0 +1,157 @@
+/**
+ * External dependencies
+ */
+import classNames from 'classnames';
+import { Icon } from '@wordpress/icons';
+import {
+	customerAccountStyle,
+	customerAccountStyleAlt,
+} from '@woocommerce/icons';
+import { InspectorControls } from '@wordpress/block-editor';
+import { __ } from '@wordpress/i18n';
+import type { BlockAttributes } from '@wordpress/blocks';
+import { getSetting } from '@woocommerce/settings';
+import { createInterpolateElement } from '@wordpress/element';
+import {
+	PanelBody,
+	SelectControl,
+	// eslint-disable-next-line @wordpress/no-unsafe-wp-apis
+	__experimentalToggleGroupControl as ToggleGroupControl,
+	// eslint-disable-next-line @wordpress/no-unsafe-wp-apis
+	__experimentalToggleGroupControlOption as ToggleGroupControlOption,
+	ExternalLink,
+} from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import { DisplayStyle, IconStyle } from './types';
+
+interface BlockSettingsProps {
+	attributes: BlockAttributes;
+	setAttributes: ( attrs: BlockAttributes ) => void;
+}
+
+const AccountSettingsLink = () => {
+	const accountSettingsUrl = `${ getSetting(
+		'adminUrl'
+	) }admin.php?page=wc-settings&tab=account`;
+
+	const linkText = createInterpolateElement(
+		`<a>${ __(
+			'Manage account settings',
+			'woo-gutenberg-products-block'
+		) }</a>`,
+		{
+			a: <ExternalLink href={ accountSettingsUrl } />,
+		}
+	);
+
+	return <div className="account-link">{ linkText }</div>;
+};
+
+export const BlockSettings = ( {
+	attributes,
+	setAttributes,
+}: BlockSettingsProps ) => {
+	const { displayStyle, iconStyle } = attributes;
+	const displayIconStyleSelector = [
+		DisplayStyle.ICON_ONLY,
+		DisplayStyle.ICON_AND_TEXT,
+	].includes( displayStyle );
+
+	return (
+		<InspectorControls key="inspector">
+			<AccountSettingsLink />
+			<PanelBody
+				title={ __( 'Display', 'woo-gutenberg-products-block' ) }
```

Comment:
It's a minor thing, but I spotted different blocks use different section titles. I started to unify filter blocks towards `Display settings`, so you may want to use the same. But I don't have a strong opinion, we may also unify other blocks to use `Display`, so leaving it up to you 😃 

```suggestion
				title={ __( 'Display settings', 'woo-gutenberg-products-block' ) }
```

Reply:
Makes sense!